CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.0] - 2024-07-31
+### Changed
+- The `hop.io.Producer` can be used to publish to more than one topic with a single instance.
+  To support this capability, the `write` method now accepts an optional `topic` argument to
```

Comment:
Is this new functionality documented in for example [the quickstart guide](https://github.com/scimma/hop-client/blob/9ddf2ca95b7063254cc7fb3a47ca101fd8ff4802/doc/user/quickstart.rst) or elsewhere?

Reply:
There is a section devoted to it: [Publishing to Multiple Topics](https://github.com/scimma/hop-client/blob/9ddf2ca95b7063254cc7fb3a47ca101fd8ff4802/doc/user/stream.rst#publishing-to-multiple-topics)